kasa/tapo/tapodevice.py

```diff
@@ -340,6 +344,13 @@ async def update_credentials(self, username: str, password: str):
         }
         return await self.protocol.query({"set_qs_info": payload})
 
+    async def set_mac(self, mac):
+        """Set the mac address.
+
+        :param str mac: mac in hexadecimal with colons, e.g. 01:23:45:67:89:ab
+        """
+        raise NotImplementedError
```

Comment:
A call to set the mac using `set_device_info` succeeds, but the mac doesn't get changed. Maybe this should raise something else than NotImplementedError.

Reply:
Yes, at least it did work on my device when I added it.